client/.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Delete this file, likely left over from resolving merge conflicts

Reply:
I deleted the file!